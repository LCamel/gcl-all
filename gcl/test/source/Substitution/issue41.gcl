con f : Int -> Bool
var i, n : Int

{: P n = ⟨ ∧ i : 0 ≤ i < n : f i ⟩ 
:}

n := n + i
{ P n }