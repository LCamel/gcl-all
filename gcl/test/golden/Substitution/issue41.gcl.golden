1: P (n + i)
  P (n + i) ===> ⟨ ∀ ?i_9 : 0 ≤ ?i_9 < n + i : f ?i_9 ⟩
    
2: P (n + i)
  P (n + i) ===> ⟨ ∀ ?i_19 : 0 ≤ ?i_19 < n + i : f ?i_19 ⟩
    